optuna/visualization/parallel_coordinate.py

```diff
@@ -7,21 +7,17 @@
 from optuna.visualization.utils import _check_plotly_availability
 from optuna.visualization.utils import is_available
 
-if type_checking.TYPE_CHECKING:
-    from typing import DefaultDict  # NOQA
-    from typing import List  # NOQA
-    from typing import Optional  # NOQA
+from typing import DefaultDict  # NOQA
+from typing import List  # NOQA
+from typing import Optional  # NOQA
+from optuna.study import Study  # NOQA
 
-    from optuna.study import Study  # NOQA
-
-if is_available():
-    from optuna.visualization.plotly_imports import go
+from optuna.visualization.plotly_imports import go
 
 logger = get_logger(__name__)
 
 
-def plot_parallel_coordinate(study, params=None):
-    # type: (Study, Optional[List[str]]) -> go.Figure
+def plot_parallel_coordinate(study: Study, params: Optional[List[str]] = None) -> go.Figure:
```

Comment:
I guess you should quote like following after reverted the checking of `is_available()`.

```suggestion
def plot_parallel_coordinate(study: Study, params: Optional[List[str]] = None) -> 'go.Figure':
```